src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -20,7 +20,7 @@ public Movie add(Movie movie) {
         try {
             session = HibernateUtil.getSessionFactory().openSession();
             transaction = session.beginTransaction();
-            session.persist(movie);
+            session.save(movie);
```

Comment:
save is deprecated, use persist instead
```suggestion
            session.persist(movie);
```